mountpoint-s3/src/build_info.rs

```diff
@@ -9,18 +9,39 @@ mod built {
 
 /// Valid SemVer version constructed using declared Cargo version and short commit hash if needed.
 pub const FULL_VERSION: &str = {
+    const BUILD_TARGET_STR: &str = match build_target_suffix() {
```

Comment:
Why match again here? Couldn't the function just return a (sometime empty) string?